packages/tabs/src/spectrum-tabs-sizes.css

```diff
@@ -11,23 +11,65 @@ governing permissions and limitations under the License.
+    --spectrum-tabs-icon-size: var(--spectrum-workflow-icon-size-50);
+    --spectrum-tabs-icon-to-text: var(--spectrum-text-to-visual-75);
+    --spectrum-tabs-top-to-icon: var(--spectrum-tab-item-top-to-workflow-icon-small);
+    --spectrum-tabs-focus-indicator-gap: var(--spectrum-tab-item-focus-indicator-gap-small);
+    --spectrum-tabs-font-size: var(--spectrum-font-size-75);
+}
+
+:host([size='l']) #list {
+    --spectrum-tabs-item-height: var(--spectrum-tab-item-height-large);
+    --spectrum-tabs-item-horizontal-spacing: var(--spectrum-tab-item-to-tab-item-horizontal-large);
+    --spectrum-tabs-item-vertical-spacing: var(--spectrum-tab-item-to-tab-item-vertical-large);
+    --spectrum-tabs-start-to-edge: var(--spectrum-tab-item-start-to-edge-large);
+    --spectrum-tabs-top-to-text: var(--spectrum-tab-item-top-to-text-large);
+    --spectrum-tabs-bottom-to-text: var(--spectrum-tab-item-bottom-to-text-large);
+    --spectrum-tabs-icon-size: var(--spectrum-workflow-icon-size-100);
+    --spectrum-tabs-icon-to-text: var(--spectrum-text-to-visual-200);
+    --spectrum-tabs-top-to-icon: var(--spectrum-tab-item-top-to-workflow-icon-large);
+    --spectrum-tabs-focus-indicator-gap: var(--spectrum-tab-item-focus-indicator-gap-large);
+    --spectrum-tabs-font-size: var(--spectrum-font-size-200);
+}
+
+:host([size='xl']) #list {
+    --spectrum-tabs-item-height: var(--spectrum-tab-item-height-extra-large);
+    --spectrum-tabs-item-horizontal-spacing: var(--spectrum-tab-item-to-tab-item-horizontal-extra-large);
+    --spectrum-tabs-item-vertical-spacing: var(--spectrum-tab-item-to-tab-item-vertical-extra-large);
+    --spectrum-tabs-start-to-edge: var(--spectrum-tab-item-start-to-edge-extra-large);
+    --spectrum-tabs-top-to-text: var(--spectrum-tab-item-top-to-text-extra-large);
+    --spectrum-tabs-bottom-to-text: var(--spectrum-tab-item-bottom-to-text-extra-large);
+    --spectrum-tabs-icon-size: var(--spectrum-workflow-icon-size-200);
+    --spectrum-tabs-icon-to-text: var(--spectrum-text-to-visual-300);
+    --spectrum-tabs-top-to-icon: var(--spectrum-tab-item-top-to-workflow-icon-extra-large);
+    --spectrum-tabs-focus-indicator-gap: var(--spectrum-tab-item-focus-indicator-gap-extra-large);
+    --spectrum-tabs-font-size: var(--spectrum-font-size-300);
+}
+
 :host([size='s']) #list.spectrum-Tabs--compact {
-    --spectrum-tabs-item-height: var(--mod-tabs-item-height-compact, var(--spectrum-tab-item-compact-height-small));
-    --spectrum-tabs-top-to-text: var(--mod-tabs-top-to-text-compact, var(--spectrum-tab-item-top-to-text-compact-small));
-    --spectrum-tabs-bottom-to-text: var(--mod-tabs-bottom-to-text-compact, var(--spectrum-tab-item-top-to-text-compact-small));
-    --spectrum-tabs-top-to-icon: var(--mod-tabs-top-to-icon-compact, var(--spectrum-tab-item-top-to-workflow-icon-compact-small));
+    --mod-tabs-item-height: var(--mod-tabs-item-height-compact, var(--spectrum-tab-item-compact-height-small));
```

Comment:
I see the same pattern of overriding mods here, let's discuss!

Reply:
Note the more specific mod in the stack: `--mod-tabs-item-height: var(--mod-tabs-item-height-compact, var(--spectrum-tab-item-compact-height-small));` - that's a very intentional override

Reply:
To override the height for compact variants, you can use `--mod-tabs-item-height-compact` at any point in the cascade.